models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -217,10 +215,10 @@ aeif_cond_beta_multisynapse::Parameters_::set( const DictionaryDatum& d )
         throw BadProperty(
           "All synaptic time constants must be strictly positive" );
       }
-      if ( tau_tmp[ i ] >= taus_decay[ i ] )
+      if ( tau_tmp[ i ] > taus_decay[ i ] )
```

Comment:
Is there any strict mathematical reason why the rise time must be shorter than the decay time? This is the usual case in neuroscience, but I think the mathematics will work perfectly well otherwise, too, so I would remove this constraint.